src/__tests__/lib/exampleData.js

```diff
@@ -207,9 +207,9 @@ export const userStatusEmojiRealm: UserStatus['status_emoji'] = deepFreeze({
 
 export const realm: URL = new URL('https://zulip.example.org');
 
-export const zulipVersion: ZulipVersion = new ZulipVersion('2.1.0-234-g7c3acf4');
-
-export const zulipFeatureLevel = 1;
+/** These may be raised but should not be lowered. */
+export const recentZulipVersion: ZulipVersion = new ZulipVersion('6.0-dev-511-g7a3abcc7ef');
+export const recentZulipFeatureLevel = 132;
```

Comment:
It'd be good for the commit message to mention what motivates increasing this. I'm guessing it's so that the FL 118 conditions in the next commit come out true?

If so, that also points to test cases that should be added in that commit that adds the conditions: it should come with a test case exercising the old-server case of each condition, confirming that we don't store the data there.